builder/assets/init_rpi.sh

```diff
@@ -38,7 +38,12 @@ echo_stamp() {
 echo_stamp "Rename SSID"
 NEW_SSID='CLEVER-'$(head -c 100 /dev/urandom | xxd -ps -c 100 | sed -e "s/[^0-9]//g" | cut -c 1-4)
 sudo sed -i.OLD "s/CLEVER/${NEW_SSID}/" /etc/wpa_supplicant/wpa_supplicant.conf
-clever_rename ${NEW_SSID}
+
+echo_stamp "Rename hostname to $NEW_SSID"
+hostnamectl set-hostname $NEW_SSID
+sed -i 's/127\.0\.1\.1.*/127.0.1.1\t'${NEW_NAME}'/g' /etc/hosts
+# make .local hostname accesable when wlan and ethernet interfaces down
+echo -e "127.0.0.1\t${NEW_NAME}.local" >> /etc/hosts
```

Comment:
Please consider using `127.0.1.1` for the `hostname.local` entry (see [Debian network setup manual](https://www.debian.org/doc/manuals/debian-reference/ch05.en.html#_the_hostname_resolution)). You could also put the `hostname.local` entry on the same line as the `hostname` (so it would be just `127.0.1.1 clever clever.local`), that would reduce the overall complexity.